src/cpp/session/modules/NotebookData.R

```diff
@@ -0,0 +1,46 @@
+#
+# NotebookData.R
+#
+# Copyright (C) 2009-16 by RStudio, Inc.
+#
+# Unless you have received this program directly from RStudio pursuant
+# to the terms of a commercial license agreement with RStudio, then
+# this program is licensed to you under the terms of version 3 of the
+# GNU Affero General Public License. This program is distributed WITHOUT
+# ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+# AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+#
+#
+
+.rs.addFunction("initDataCapture", function(outputFolder, libraryFolder)
+{
+  assign("print.data.frame", function(x, ...) {
```

Comment:
I think if someone goes to the trouble of creating a `tbl_df` we should respect it's default print method (as that might have been one of the motivators for creating it in the first place).